lib/XML/LibXML/CStructs.pm

```diff
@@ -2,11 +2,15 @@ use v6;
 use nqp;
 use NativeCall;
 
+constant XML_XML_NAMESPACE = "http://www.w3.org/XML/1998/namespace";
+
+my $LIBXML_VERSION := cglobal('libc.so.6', 'LIBXML_VERSION', int32);
```

Comment:
Actually, that _doesn't work_ because libxml2 does not have a function that expressly returns the version. They have routines for the parser version (maybe that would work) and the SAX version, but not the library version.

LIBXML_VERSION is actually a macro... which doesn't help us in Nativecall.